### ISSUE TITLE
Releases should include binaries

### ISSUE THREAD

**Issue by Hoverbear** [tikv/tikv#4426]
When making a release on https://github.com/tikv/tikv/releases we currently only release source code.

For users interested in using TiKV directly, this means they need to download TiDB and extract TiKV, or build TiKV themself. It would be highly advantageous for them to be able to easily download the released portable build.

I am fairly sure these already exist somewhere for every version. Do you know where they are @zhangjinpeng1987 ? I'd be happy to go and do this if I happened to know where they binaries are, or if I needed to build them all.

**Comment by BusyJay** [comment]
I think it's part of product strategy to only provide deploy solution instead of individual binaries. Deploy solution can be found in https://pingcap.com/docs/QUICKSTART/.

**Comment by Hoverbear** [comment]
Okay. I'll work to provide some better access to tools like `tikv-ctl` via docker containers then so we can easily integrate them

**Comment by brson** [comment]
I agree that since TiKV is a standalone project that it should have its own binary releases. It's not obvious to me why we it would be strategic to not publish plain binaries in addition to docker containers, etc.

**Comment by BusyJay** [comment]
There are also [deployment steps](https://github.com/tikv/tikv/#testing-deployment) in README file that shows how to deploy with pure binary files.

**Comment by Hoverbear** [comment]
These steps instruct the user to download the `latest` TiDB package from PingCAP.

This ticket explicitly requests per-release downloads available through Github releases, not just source code.

You can see this is a common practice, especially among other CNCF projects:

* https://github.com/kubernetes/kubernetes/releases
* https://github.com/open-policy-agent/opa/releases
* https://github.com/prometheus/prometheus/releases
* https://github.com/linkerd/linkerd/releases
* https://github.com/coredns/coredns/releases
* https://github.com/containerd/containerd/releases
